app/dashboard/organization/overview/page.tsx

```diff
@@ -1,11 +1,213 @@
+                    <div className="2xl:text-4xl text-3xl 2xl:mb-10 mb-4">Organization Info</div>
+
+                    <div className="text-body mb-1">Organization Name*</div>
+                    {isOwner ?
+                    <input className="bg-transparent border-b border-heading focus:outline-none w-3/5" value={orgName} onChange={e => setOrgName(e.target.value)}/>
+                    :
+                    <div>{orgName}</div>
+                    }
+                    {formError && orgName==="" && <div className="text-red-800 mt-1 2xl:text-base text-sm">Field required</div>}
 
+                    <div className="text-body mb-1 2xl:mt-10 mt-8">Organization Email*</div>
+                    {isOwner ?
+                    <input className="bg-transparent border-b border-heading focus:outline-none w-3/5" value={orgEmail} onChange={e => setOrgEmail(e.target.value)}/>
+                    :
+                    <div>{orgEmail}</div>
+                    }
+                    
+                    {formError && orgEmail==="" && <div className="text-red-800 mt-1 2xl:text-base text-sm">Field required</div>}
+
+                    <div className="text-body mb-1 2xl:mt-10 mt-8">Organization Website</div>
+                    {isOwner ?
+                    <input className="bg-transparent border-b border-heading focus:outline-none w-3/5" value={orgWebsite} onChange={e => setOrgWebsite(e.target.value)}/>
+                    :
+                    <div>{orgWebsite}</div>
+                    }
+
+                    <div className="text-body mb-1 2xl:mt-10 mt-8">Organization Bio</div>
+                    {isOwner ?
+                    <textarea className="bg-transparent border-b border-heading focus:outline-none w-3/5 min-h-[5rem]" value={orgBio} onChange={e => setOrgBio(e.target.value)}/>
+                    :
+                    <div className="w-3/5 min-h-[5rem]">{orgBio}</div>
+                    }
+                    <div className="text-body mt-6">You are</div>
+                    <div>{orgData?.isNonProfit ? "Receiving Funding" : "Giving Funding"}</div>
+
+                    {
+                    isOwner &&
+                    <button className="block px-3 py-1 border-2 border-darkmaroon text-darkmaroon 2xl:text-xl text-lg rounded-xl 2xl:mt-14 mt-6 hover:bg-darkmaroon/20 transition duration-300" onClick={() => handleUpdateOrg()}>{
+                        !loading ? "Update Info" : "Updating Info..."}
+                    </button>
+                    }
+
+                </div>
+                {/* Team Section */}
+                <OrgTeam orgTeam={orgTeam || []} isOwner = {isOwner || false}/>
+            </div>
+            {/* End Org info section */}
+            <div className="mt-24">
+                <div className="text-2xl">Danger Zone</div>
+                <div className="w-96 border-b-[1px] border-body mt-1 mb-3"></div>
```

Comment:
Little nitpick that we can include in the next pr or just ignore. I think it looks better if we change mb-3 to mb-5
![image](https://github.com/nam-t24/Senior-CS-Project/assets/70723826/4629b0ff-b3cc-4138-b134-01f630597fd9)
![image](https://github.com/nam-t24/Senior-CS-Project/assets/70723826/eb9fa2c8-ae4d-4674-880b-ed458747afc5)

Reply:
will do wizard nd